uisdk/src/main/java/com/karhoo/uisdk/screen/booking/BookingActivity.kt

```diff
@@ -289,7 +291,8 @@ class BookingActivity : BaseActivity(), AddressBarMVP.Actions, BookingMapMVP.Act
     }
 
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        if (requestCode == REQ_CODE_BRAINTREE || requestCode == REQ_CODE_BRAINTREE_GUEST) {
+        if (requestCode == REQ_CODE_BRAINTREE || requestCode == REQ_CODE_BRAINTREE_GUEST ||
+                requestCode == DROP_IN_REQUEST_CODE) {
```

Comment:
I wonder if we should create a local variable REQ_CODE_ADYEN (similar to REQ_CODE_BRAINTREE) and assign the value DROP_IN_REQUEST_CODE to it? It would help us to remove the reference to Adyen library in this activity and hide it behind our abstraction

Reply:
It's actually set by Adyen and not us so not sure about that in case they decide to change it at some point? 🤔

Reply:
I am suggesting to have a 

```
class AdyenPaymentView : PaymentDropInMVP.View {
...
    companion object {
        private const val REQ_CODE_ADYEN = DropIn.Companion.DROP_IN_REQUEST_CODE
    }
}
```

Reply:
Ah! Okay. I get you now.